bin/commands/new.js

```diff
@@ -3,77 +3,43 @@
  * https://github.com/insin/nwb/blob/v0.22.0/src/createProject.js
  */
 const path = require('path');
-const chalk = require('chalk');
-const copyTemplateDir = require('copy-template-dir');
 const runSeries = require('run-series');
+const semver = require('semver');
 const install = require('../util/install');
 const pkg = require('../../package.json');
 const { initGit, initialCommit } = require('../util/git');
 const inquirer = require('inquirer');
+const copyTemplate = require('../util/copy-template');
 
-const STABLE_VERSIONS = {
-    // dependencies
-    'prop-types': '15.7.2',
-    'styled-components': '5.2.2',
-    // devDependencies
-    'babel-plugin-styled-components': '1.12.0',
-    'babel-preset-zillow': '4.4.0',
-    'eslint-plugin-jest': '24.3.2',
-    'eslint-plugin-zillow': '4.0.0',
-    'husky': '4.3.8',
-    'jest-styled-components': '7.0.3',
-    'react': '17.0.2',
-    'react-dom': '17.0.2',
-    'react-test-renderer': '17.0.2',
-    // Always use the latest version of create-react-styleguide
-    'create-react-styleguide': '',
+const STABLE_DEPENDENCIES = {
+    'prop-types': pkg.devDependencies['prop-types'],
+    'react': pkg.devDependencies.react,
+    'react-dom': pkg.devDependencies['react-dom'],
+    'styled-components': pkg.devDependencies['styled-components'],
 };
 
-/**
- * Copy a project template and log created files if successful.
- */
-function copyTemplate(templateDir, targetDir, templateVars, cb) {
-    copyTemplateDir(templateDir, targetDir, templateVars, (err, createdFiles) => {
-        if (err) {
-            cb(err);
-            return;
-        }
-        createdFiles.sort().forEach(createdFile => {
-            const relativePath = path.relative(targetDir, createdFile);
-            // eslint-disable-next-line no-console
-            console.log(`  ${chalk.green('create')} ${relativePath}`);
-        });
-        cb();
-    });
-}
+const STABLE_DEV_DEPENDENCIES = {
+    'create-react-styleguide': pkg.version,
+    'husky': '4.3.8',
```

Comment:
I'm leaving Husky as is since v7 has a lot of changes and it will require a separate MR.

Reply:
Yeah, makes sense! I've been procrastinating on that 😆